src/system/term/mod.rs

```diff
@@ -65,7 +65,7 @@ impl Pty {
         Ok(Self {
             path,
             leader: PtyLeader {
-                // SAFETY: `leader` has been set by `openpty`
+                // SAFETY: `openpty` has set `leader` to an open fd suitable for assuming ownership by `OwnedFd`.
```

Comment:
This change should also apply to the OwnedFd::from_raw_fd for the file field.

Reply:
You mean for the `follower` thing right below it right?

Reply:
Yes